qiime/core/type/signature.py

```diff
@@ -133,12 +146,12 @@ def _parse_signature(self, callable, inputs, parameters, outputs):
                                     " parameters in callable signature.")
                 annotated_inputs[name] = ParameterSpec(
                     qiime_type=inputs.pop(name), view_type=view_type,
-                    default=default)
+                    default=default, help_text=help_text.get(name))
```

Comment:
This should be `help_text=help_text.get(name, Signature.NOVALUE)` to be consistent with other elements. (Currently will be set to `None` if not present).